classes/Form/ConfigurationForm.php

```diff
@@ -112,7 +113,7 @@ public function generate()
                     'type' => 'switch',
                     'label' => $this->module->getTranslator()->trans('Anonymize IP', [], 'Modules.Googleanalytics.Admin'),
                     'name' => 'GA_ANONYMIZE_ENABLED',
-                    'hint' => $this->module->getTranslator()->trans('Use this option to anonymize the visitor’s IP to comply with data privacy laws in some countries', [], 'Modules.Googleanalytics.Admin'),
+                    'desc' => $this->module->getTranslator()->trans('Use this option to anonymize the visitor’s IP to comply with data privacy laws in some countries', [], 'Modules.Googleanalytics.Admin'),
```

Comment:
I think we can use `$this->module->trans` instead.

Reply:
@leemyongpakvn It didn't work for some reason, that's why I used `$this->module->getTranslator()`. 🤷‍♂️
See https://github.com/PrestaShop/ps_googleanalytics/pull/148#pullrequestreview-1474916493